### PR TITLE
QOL: Automatically separate composer and vocals

### PR DESCRIPTION
**Pull request by Khronophobia**
prbeforeafter

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -173,7 +173,9 @@ func origMain(isOptionSpecified bool) {
 
 	fmt.Print("exoファイルを生成中... ")
 
-	artists := fmt.Sprintf("作詞：？    作曲：%s    編曲：？\r\nVo：？   譜面作成：%s", chart.Artists, chart.Author)
+	artists_slice := strings.Split(chart.Artists, " / ")
```
Comment (sevenc-nanashi) [inline main.go:176]: [Potato Leaves](https://github.com/sevenc-nanashi/potato_leaves) does not always have `/` in artists, so it needs fallback.

**Review by sevenc-nanashi — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (Khronophobia) [reply main.go]: Thanks for catching that. Is it ok now?

Comment (Khronophobia) [reply main.go]: Right now it checks if there's ` / ` in artists, but would it be better if it checks whether the chart comes from Chart Cyanvas or Potato Leaves?

Comment (Khronophobia) [reply main.go]: I changed it so that it only separates the composer and vocals if the chart is from Chart Cyanvas. Let me know what you think

**Review by sevenc-nanashi — APPROVED** [review]
LGTM, thank you for contribution!

**Pull request merged**